server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailQueryMethodContract.scala

```diff
@@ -522,15 +522,15 @@ trait EmailQueryMethodContract {
           .build(message))
       .getMessageId
 
-    val requestDateMessage2 = Date.from(ZonedDateTime.now().minusDays(1).toInstant)
+    val requestDateMessage2 = Date.from(ZonedDateTime.now().minusDays(2).toInstant)
     val messageId2 = server.getProbe(classOf[MailboxProbeImpl])
       .appendMessage(BOB.asString, MailboxPath.inbox(BOB),
         AppendCommand.builder()
           .withInternalDate(requestDateMessage2)
           .build(message))
       .getMessageId
 
-    val requestDateMessage3 = Date.from(ZonedDateTime.now().minusDays(1).toInstant)
+    val requestDateMessage3 = Date.from(ZonedDateTime.now().minusDays(3).toInstant)
```

Comment:
idem you inversed the order of element by sent at. But did not change the assertion